makefile.shade

```diff
@@ -0,0 +1,28 @@
+
+var VersionPrefix = '1.2.0'
+var VERSION='0.1'
+var FULL_VERSION='0.1'
+use-standard-lifecycle
+k-standard-goals
+
+#pack-sources target='compile'
```

Comment:
Would it make sense to make this a part of KoreBuild? 

`if (Directory.Exists("shared") { Enumerate subdirs + pack }`